pkg/cmd/pipelinerun/log_reader.go

```diff
@@ -90,12 +92,18 @@ func (lr *LogReader) readLiveLogs(pr *v1alpha1.PipelineRun) (<-chan Log, <-chan
 		}
 
 		wg.Wait()
+
+		if pr.Status.Conditions[0].Status == corev1.ConditionFalse {
```

Comment:
nit: for later on, we always make the assumption thet `pr.Status.Conditions[0]` is the `Succeeded` condition. This is the case today, this might not be the case in the future. Nothing to fix in this PR but we will need to fix that someday :angel: